### PR TITLE
Draft-023

### PR DESCRIPTION
**Pull request by ESTHER0533**
@deletosh Add draft 023

### CODE REVIEW

File: draft-023.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
+## Brief #023: Law Firm Innovation | Finding Client Development Opportunity in Crisis
+
+### How to Build A Law Firm Brand, How to Use Sales Battle Cards to Close More Deals, 
+
+## This Week's Favorite
+
+[Law Firm Innovation | Finding Client Development Opportunity in Crisis](https://www.attorneyatwork.com/law-firm-client-development-opportunity-in-crisis-law-firm-innovation/) 
+
+Businesses around the world continue to feel the pandemic’s impact in countless ways. Most in-house legal departments still grapple with the challenges of working remotely, making do with leaner budgets and resources, and managing virtual teams — all during an unprecedented era across nearly all industries and geographies. Similarly, many private clients struggle to overcome hurdles related to limited access to technology or lack of funds. Times of crisis and uncertainty always present opportunity.......
```
Comment (deletosh) [inline draft-023.md:10]: @ESTHER0533 let's end this (and all the other sections) with an entising lead that encourages them to click into the article.

File: draft-023.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+[Law Firm Content Marketing: 5 Solutions to Common Obstacles](https://fsquaredmarketing.com/law-firm-content-marketing-solutions-to-common-obstacles/)
+
+Content marketing is a wildly popular strategy and one that won’t be going away anytime soon. According to a 2020 study by HubSpot, a marketing software company, 70% of marketers are actively investing in content marketing. There are several reasons why content marketing is a smart strategy for law firms:
+
+
+----
+
+## Industry
+
+[Will artificial intelligence replace lawyers? - MikeLegal](https://blog.mikelegal.com/legaltech/will-artificial-intelligence-replace-lawyers/)
```
Comment (deletosh) [inline draft-023.md:31]: @ESTHER0533 exclude the name of the source in the tilte, e.g. in this "MikeLegal" and same for the others

**Review by deletosh — CHANGES_REQUESTED** [review]
@ESTHER0533 please see notes in the review and update

File: draft-023.md (hunk 2 above)
Comment (ESTHER0533) [reply draft-023.md:31]: @deletosh please throw more light on this, i dont understand this part

File: draft-023.md (hunk 1 above)
Comment (ESTHER0533) [reply draft-023.md:10]: @deletosh do you mean, (and all the other sections) should come after opportunity??

File: draft-023.md (hunk 2 above)
Comment (deletosh) [reply draft-023.md]: @ESTHER0533  don't inlude "MikeLegal"

**Pull request merged**